src/harness/fake_sas.py

```diff
@@ -176,7 +176,15 @@ def _GetSuccessResponse(self):
   def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
-
+  def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
+    zoneData = req['zoneDate']
+    if(zoneData['usage'] == 'EXCLUSION_ZONE'):
+      idAttributes = zoneData['id'].split('/')
+      zoneId = ''; 
+     for zoneIdPart in idAttributes[4 : len(idAttributes)]:
+       zoneId+=zoneIdPart
+     return {'zone_id' : zoneId}
```

Comment:
What happens is usage isn't 'EXCLUSION_ZONE'? zoneId won't be defined and this will cause an error. We probably want to return an HTTP error in this case (or just ignore the problem and return a string anyway).